sdk/core/Azure.Core/src/Pipeline/HttpPipelineMessage.cs

```diff
@@ -44,7 +42,9 @@ public Response Response
 
         public bool HasResponse => _response != null;
 
-        public ResponseClassifier ResponseClassifier { get; }
+        public CancellationToken CancellationToken { get; set; }
```

Comment:
Should it be null able?

Reply:
Nah, CancellationToken is a struct and `default` value does the right thing.